### ISSUE TITLE
Hutch-python Engineering Mode

### ISSUE THREAD

**Issue by ZryletTC** [pcdshub/xcs#5]
So, in general, engineering mode is on by default, which makes sense.

However, for hutch-python, since it is often run by scientists and users, should we turn engineering mode off, so that the tab-complete is more manageable for them? We can always turn it back on when we want to.

Thoughts? @ZLLentz @silkenelson @rajanplumley

**Comment by ZLLentz** [comment]
The intention was to do it in the main `hutch-python` repo as to enable it for all hutches

**Comment by silkenelson** [comment]
Is there a way that I can toggle this in the session? That would be convenient (at least for me).

**Comment by ZryletTC** [comment]
Yes, you can run `pcdsdevices.interface.set_engineering_mode(False)` to enable whitelisting. Or True to turn it back off.
